proto/ptypes/controller.proto

```diff
@@ -136,14 +136,15 @@ message VersionDetailGetReply {
     VersionOutput version = 1;
 }
 
-message Metric {
-    uint64 readBandwidth = 1;
-    uint64 writeBandwidth = 2;
+message Metrics {
+    uint64 readThroughput = 1;
+    uint64 writeThroughput = 2;
     uint64 readLatency = 3;
     uint64 writeLatency = 4;
-    uint64 iOPS = 5;
+    uint64 readIOPS = 5;
+    uint64 writeIOPS = 6;
 }
```

Comment:
It's ok to change the properties, because metrics does not be used before. (literally, we should not change existing fields but rather add new ones. )